wasm-resources/index.em.template.html

```diff
@@ -0,0 +1,1300 @@
+<!doctype html>
```

Comment:
can this template be reduced, or is this a standard one?

Reply:
This is standard one, however I can reduce it to simple one.

Reply:
would be great!